From c954d7d224fe7c60b265bb8d9403b97d7b5ca207 Mon Sep 17 00:00:00 2001
From: Jan Sowinski <jan.sowinski@intel.com>
Date: Fri, 9 Dec 2022 13:47:56 +0100
Subject: [PATCH] disable zlib and gzip

zlib is not required for current configuration and has vulnerabilities.
Hence disabling zlib along with gzip which requires it.

Signed-off-by: Jan Sowinski <jan.sowinski@intel.com>

---
 lib/Kconfig | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/lib/Kconfig b/lib/Kconfig
index 3c5a4ab386..ea2650ce2f 100644
--- a/lib/Kconfig
+++ b/lib/Kconfig
@@ -659,7 +659,7 @@ config LZO
 config GZIP
 	bool "Enable gzip decompression support"
 	select ZLIB
-	default y
+	default n
 	help
 	  This enables support for GZIP compression algorithm.
 
@@ -680,8 +680,8 @@ config BZIP2
 	  This enables support for BZIP2 compression algorithm.
 
 config ZLIB
-	bool
-	default y
+	bool "Enable ZLIB compression lib"
+	default n
 	help
 	  This enables ZLIB compression lib.
 
-- 
2.25.1

